core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -2,13 +2,16 @@ package forwarders
 
 import (
 	"context"
+	"slices"
 	"sync"
 	"time"
 
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/graph-gophers/graphql-go/errors"
```

Comment:
Is this an auto-import mistake?
```suggestion
	"errors"
```